04.word2vec/train.py

```diff
@@ -18,6 +20,26 @@ def wordemb(inlayer):
     return wordemb
 
 
+# save and load word dict and embedding table
+def save_dict_and_embedding(word_dict, embeddings):
+    with open("word_dict", "w") as f:
+        for key in word_dict:
+            f.write(key + " " + str(word_dict[key]) + "\n")
+    with open("embedding_table", "w") as f:
+        numpy.savetxt(f, embeddings, delimiter=',', newline='\n')
+
+
+def load_dict_and_embedding():
+    word_dict = dict()
+    with open("word_dict", "r") as f:
+        for line in f:
+            key, value = line.strip().split(" ")
```

Comment:
Maybe save in `csv` is a better idea, otherwise, the word cannot contain spaces.

It could be true since we could make some phrases as in our word dict.

Reply:
cool, will do this later